app/src/test/java/nu/yona/app/utils/MobileNumberTests.java

```diff
@@ -0,0 +1,85 @@
+package nu.yona.app.utils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MobileNumberTests {
```

Comment:
Please rename this into ``MobileNumberFormatterTest``: The name of the class under test, followed by ``Test``.